src/game/__tests__/eop.test.js

```diff
@@ -1,12 +1,13 @@
 import { Client } from 'boardgame.io/client';
 import { DEFAULT_START_SUIT, INVALID_CARDS, STARTING_CARD_MAP } from '../../utils/constants';
 import { ElevationOfPrivilege } from '../eop';
+import { Local } from 'boardgame.io/multiplayer'
 
 describe('game', () => {
   const spec = {
     game: ElevationOfPrivilege,
     numPlayers: 3,
-    multiplayer: { local: true },
+    multiplayer: Local(),
```

Comment:
What is `Local()`?  Maybe add a link to the documentation?

Reply:
Do you want me to link it as a comment in the code or just here?

It's similar to the new `SocketIO({})` bit in app.js

It's a factory function which returns a new `Transport` object. I they changed this to allow you to write your own code to interact with the server. The documentation doesn't really explain this very well or at all actually, but here's the links to the relevant parts.

- [this](https://boardgame.io/documentation/#/multiplayer) Talks more about local and remote masters
- [this](https://boardgame.io/documentation/#/api/Client) explains setting the value in the client